packages/core/src/RenderingEngine/RenderingEngine.ts

```diff
@@ -210,6 +210,7 @@ class RenderingEngine implements IRenderingEngine {
 
     // 5. Remove the requested viewport from the rendering engine
     this._removeViewport(viewportId);
+    viewport.hasBeenDisabled = true;
```

Comment:
The naming here is very odd, and I think it ends up not being quite right.  hasBeenDisabled doesn't mean it is currently disabled, but suggests that at some point in the past it got disabled.   Is it safe to say:
viewport.isDisabled = true 
or
viewport.disabled = true
Otherwise, what about making it more prescriptive, something like disableOverlayDrawing, so that it is specifying that no overlays are to be drawn this viewport, and that would include things like the crosshairs, but may also include things like segmentation overlays etc.

Reply:
Makes sense